eddy/core/items/nodes/common/label.py

```diff
@@ -100,6 +101,23 @@ def updatePos(self, moved=False):
         if not moved:
             self.setPos(self.defaultPos())
 
+    def wrapLabel(self):
+        """
+        Wrap label into node.
+        """
+        super().wrapLabel()
+        if self.parentObject().isNode() and self.parentObject().type() in [Item.ConceptNode,
```

Comment:
Un paio di osservazioni:
1) Meglio usare `QGraphicsItem.parentItem()` per ottenere il riferimento all'item genitore in quanto non è detto che tutti gli item siano oggetti. Nel nostro caso è vero per i nodi entità ma vorrei cambiarlo in futuro, e questo causerebbe problemi con questa implmentazione.
2) L'implementazione di per se va bene, non sono però sicuro `NodeLabel` sia il punto migliore, sto cercando di capire se è meglio spostarla.

Reply:
Per il punto 1 non c'è problema, ora lo sostituisco. 
Per quanto riguarda il punto migliore per la `wrapLabel` in effetti ho avuto qualche dubbio anch'io ma alla fine mi sembrava la posizione migliore.